lib/rust/ensogl/core/src/animation/frp/animation.rs

```diff
@@ -29,23 +29,28 @@ pub mod hysteretic;
 //     }
 //     Output {
 //         value  (f32),
-//         on_end (inertia::EndStatus),
+//         on_end (),
 //     }
 // }
 
 /// Simulator used to run the animation.
 pub type AnimationSimulator<T> = inertia::DynSimulator<mix::Repr<T>>;
 
+/// Default animation precision.
+pub const DEFAULT_PRECISION: f32 = 0.001;
+
```

Comment:
This doc does not explain what it is at all. After reading it, I still dont understand what it does.

Reply:
Added additional docs